packages/cli/src/commands/init/init.ts

```diff
@@ -307,6 +309,27 @@ function userAgentPackageManager() {
   return null;
 }
 
+const createGitRepository = async (folder: string) => {
```

Comment:
This is doing more than creating for repo. Do we really want to make initial commit? What about systems where fit is not available? We’re not making any checks, so this adds extra surface for init errors

Reply:
If Git won't be available I think it would be good to simply add log to inform that there's a problem with creating Git repo and just continue (and I think that we can log error with `logger.debug` so that advanced users can use check it with `--verbose`).

Reply:
Yeah, let’s just make sure we handle the case

Reply:
done ✅ I also added handling case when somebody tries to init project inside already existing git repo.